frontend/src/pages/TrainingReports/components/EventCards.js

```diff
@@ -1,25 +1,43 @@
 /* eslint-disable jsx-a11y/anchor-is-valid */
 import React from 'react';
 import PropTypes from 'prop-types';
-import { eventPropTypes } from '../constants';
+import { eventPropTypes, EVENT_STATUS } from '../constants';

```

Comment:
now that `@ttahub/common` is a published package, we should probably make use of it for things like this

not a blocker for this PR, but sometime soon we should have a PR that removes the local `@ttahub/common` package in favor of the published version and replace these import statements